internal/services/permissions/permissions.go

```diff
@@ -13,8 +13,8 @@ import (
 
 var permErrorFmt = "this action is only allowed for root users. Current user is %d"
 
-// Authorizer is an abstraction of authorization process.
-type Authorizer struct {
+// PermissionManager is an abstraction of permission process.
+type PermissionManager struct {
```

Comment:
This is already part of the `permissions` package, so it should only be `Manager`, otherwise we are looking at `permissions.PermissionManager` (not ideal)

Reply:
waow, well spotted!